src/FTPC.jl

```diff
@@ -113,7 +115,7 @@ end
 # Callbacks
 ##############################
 
-function write_file_cb(buff::Ptr{UInt8}, sz::Csize_t, n::Csize_t, p_wd::Ptr{Void})
+function write_file_cb(buff::Ptr{UInt8}, sz::Csize_t, n::Csize_t, p_wd::Ptr{Nothing})
```

Comment:
Since you're just C calls here you should be using `Ptr{Cvoid}` instead of `Ptr{Nothing}`. Details in the [Julia NEWS](https://github.com/JuliaLang/julia/blob/master/NEWS.md)

> `Void` has been renamed back to `Nothing` with an alias `Cvoid` for use when calling C with a return type of `Cvoid` or a return or argument type of `Ptr{Cvoid}` ([#25162](https://github.com/JuliaLang/julia/issues/25162)).

Reply:
Fixed